src/containers/App/App.css

```diff
@@ -1,28 +1,14 @@
 .App {
   text-align: center;
-  background-color: #6b6b6b;
-  width: fit-content;
-}
-
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
 }
 
 .App-header {
-  background-color: #222;
-  height: 150px;
-  padding: 20px;
+  display: flex;
```

Comment:
no flex please use grid. Best to use one or the other but not both and I prefer grid, it's more declarative.

Reply:
Sure, I just used flex here because it was the easiest solution to getting elements laid out in a single dimension and having the empty space flex to expand to keep the logo in the left and login button on the right, while having the content of each element determine their widths. It is kinda avoiding declaring the layout explicitly though.

Should be able to switch it over to:
```
  display: grid;
  grid-template-columns: repeat(3, auto);
  justify-content: space-between;
```